src/multi_vector_simulator/D2_model_constraints.py

```diff
@@ -179,10 +185,10 @@ def constraint_minimal_renewable_share(model, dict_values, dict_model):
     renewable_assets, non_renewable_assets = prepare_constraint_minimal_renewable_share(
         dict_values,
         dict_model,
-        oemof_solph_object_asset,
-        weighting_factor_energy_carrier,
-        renewable_share_asset_flow,
-        oemof_solph_object_bus,
+        OEMOF_SOLPH_OBJECT_ASSET,
+        WEIGHTING_FACTOR_ENERGY_CARRIER,
+        RENEWABLE_SHARE_ASSET_FLOW,
+        OEMOF_SOLPH_OBJECT_BUS,
```

Comment:
As the variables are now constants at the top of `D2`, you don´t need to use them as arguments in the function anympre - you can just delete them.
```suggestion
```
Same then in `def prepare_constraint_minimal_renewable_share(...)` though and its docstrings.

Reply:
Sure, I will do this everywhere